ssg/build_cpe.py

```diff
@@ -104,6 +106,46 @@ def get_cpe_name(self, cpe_id):
     def get_product_cpe_names(self):
         return [ cpe.name for cpe in self.product_cpes.values() ]
 
+    def parse_platform_definition(self, platforms):
+        # let's construct the platform id
+        id_list = [self._convert_platform_to_id(platform) for platform in platforms]
+        id = "_or_".join(id_list)
```

Comment:
This is "fusing" multiple 'OR'ed platform entries into a single `CPEPlatform`, but I think it should create a `CPEPlatform` per entry in`platforms`.

I think merging 'OR'ed platforms into a single 'CPEPlatform' will reduce their reusability and increase the number of unique CPEALPlatforms in the content.

For example:
```
Rule AB.yml
platforms:
  - Expression A
  - Expression B

Rule AC.yml
platforms:
  - Expression A
  - Expression C

Rule BC.yml
platforms:
  - Expression B
  - Expression C

Rule ABC.yml
platforms:
  - Expression A
  - Expression B
  - Expression C
```

When each rule gets a `CPEPlatform` that includes the 'OR'ed platforms, each rule may get a unique complex `logical-test'. From the example above we would get 4 CPEALPlatforms. (Each rule may end up having its own unique platform).

If we make a `CPEPlatform` from each entry in the `platforms` key, we get much smaller `CPEALPlatforms` that can be used by multiple rules.
From the example, we would get  3 CPEALPlatforms, one for each Expression.